.rustfmt.toml

```diff
@@ -0,0 +1 @@
+use_field_init_shorthand = true
```

Comment:
why is this needed?

Reply:
It's not needed, but it means that the some of the refactoring I did (using "field init shorthand") won't need to be done again, since this can be handled automatically by rustfmt

Reply:
I see, sounds good